{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "d39d825d_1f7fd696",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 3
      },
      "lineNbr": 0,
      "author": {
        "id": 1021694
      },
      "writtenOn": "2024-08-05T10:41:21Z",
      "side": 1,
      "message": "PTAL. I\u0027m unable to start the trybot in this repo.",
      "revId": "265071e9463dd29cfd82d201b88b274fab9fa918",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735"
    }
  ]
}